### PR TITLE
Create mcsrawrecordingextractor.py

### PR DESCRIPTION
**Pull request by abhijithkrishnan93**
This will allow to extract data from mcs_raw_binary (.raw) files, which is converted from .mcd file using MC_Data_Tool.

### CODE REVIEW

File: spikeextractors/extractors/neoextractors/mcsrawrecordingextractor.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from .neobaseextractor import NeoBaseRecordingExtractor, NeoBaseSortingExtractor
+
+try:
+    import neo
+    HAVE_NEO = True
+except ImportError:
+    HAVE_NEO = False
+ 
+
+class mcsrawRecordingExtractor(NeoBaseRecordingExtractor):
```
Comment (samuelgarcia) [inline spikeextractors/extractors/neoextractors/mcsrawrecordingextractor.py:10]: I would name it `MCSRawRecordingExtractor` or `McsRawRecordingExtractor`

**Comment by samuelgarcia** [comment]
Thnak you.
Except the class name. I think this is OK.

**Comment by abhijithkrishnan93** [comment]
Yes. I think. I should close this now?

**Comment by alejoe91** [comment]
No don't close it! We have to merge it :) 

You have to change the `__init__` to include this new extractor. As it stands, the extractor is not available to the user.

**Comment by abhijithkrishnan93** [comment]
Okk. Sorry. :)  I will not close it

**Comment by alejoe91** [comment]
@samuelgarcia this is ready to merge. Do you have a test MCSRaw file to test?

**Pull request merged**